lib/cloud/asset.js

```diff
@@ -47,3 +53,150 @@ export function staticAssetHandler(req) {
     body: data
   });
 }
+
+class Signer {
+  sign() {
```

Comment:
The sign interface is designed to return a `Promise`? May be wee some comment on this.